tests/helper/helper_generic.go

```diff
@@ -341,10 +341,12 @@ func CommonBeforeEach() CommonVar {
 	commonVar.Project = commonVar.CliRunner.CreateAndSetRandNamespaceProject()
 	commonVar.OriginalWorkingDirectory = Getwd()
 	os.Setenv("GLOBALODOCONFIG", filepath.Join(commonVar.ConfigDir, "preference.yaml"))
-	// Set ConsentTelemetry to false so that it does not prompt to set a preference value
+	// Set ConsentTelemetry to true so that it does not prompt to set a preference value
```

Comment:
But the code says we are setting it to false

Reply:
oops left over from addressing comments